cantabular/queries.go

```diff
@@ -247,6 +245,33 @@ query ($dataset: String!, $text: String!, $category: String!, $limit: Int!, $off
   }
 `
 
+// QueryAreasWithoutPagination is the graphQL query to search for areas and area types which match a specific string.
+const QueryAreasWithoutPagination = `
```

Comment:
a lot of the changes above are about pagination so slightly confused about the existence of this query